Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM golang:1.17.6 as builder
+
+# Clone repositories
+WORKDIR /repos
+RUN git clone https://github.com/ElrondNetwork/rosetta-docker-scripts.git --branch=rc/v1 --depth=1
+RUN git clone https://github.com/ElrondNetwork/elrond-config-devnet --branch=rc-2022-july --depth=1
+RUN git clone https://github.com/ElrondNetwork/elrond-config-mainnet --branch=rc-2022-july --depth=1
+WORKDIR /go
+RUN git clone https://github.com/ElrondNetwork/elrond-go.git --branch=rc/2022-july --depth=1
+RUN git clone https://github.com/ElrondNetwork/rosetta.git --branch=main --depth=1
+
+# Build rosetta
+WORKDIR /go/rosetta/cmd/rosetta
+RUN go build
+
+# Build node
+WORKDIR /go/elrond-go/cmd/node
+RUN go build -i -v -ldflags="-X main.appVersion=rc/2022-july"
```

Comment:
let's not hardcode the node's version but instead to take it from git, somehow

Reply:
Fixed.